source/settings/config-file.txt

```diff
@@ -108,6 +108,22 @@ Settings
    * - :ref:`protectConnectionStrings <compass-protect-connection-strings>`
      - .. include:: /includes/fact-protect-connection-strings.rst 
 
+   * - protectConnectionStringsForNewConnections
+     - Hides credentials in your connection string for new connections. 
+
+       *Default:* false
+
+       If ``protectConnectionStringsForNewConnections`` is ``true``,
+       the default state of the :guilabel:`Edit connection string`
+       option in the :guilabel:`General` connection tab is off for new
+       connections.
+
+       .. note:: 
+
+          Configuring the ``protectConnectionStringsForNewConnections``
+          setting does not prevent users from manually toggling the
+          :guilabel:`Edit connection string` option on or off. 
```

Comment:
[suggestion] It might be more effective here to say that "the settings doesn't prevent users from manually **enabling** the `Edit connection string` option" because that's the specific circumstance that users could potentially be concerned about and they might already expect for it to be off. Also, it could help simplify the language since (on quick glance) it looks like we typically refer to toggles as a noun.

Reply:
Done! Thanks for your feedback!